presto-native-execution/presto_cpp/main/SystemConnector.h

```diff
@@ -0,0 +1,206 @@
+/*
```

Comment:
We can may be add a `connectors` folder under `presto_cpp`

Reply:
@majetideepak : I had given this a thought. 

But since this code depends on TaskManager, that could introduce a circular dependency between main and connectors. So this connector is a good fit for the main folder. Other connectors in the future could be moved to a connectors folder.